.golangci.yml

```diff
@@ -1,19 +1,30 @@
+version: "2"
 run:
-  # Our CI expects a vendor directory, hence we have to use -mod=readonly here as well.
   modules-download-mode: readonly
-  deadline: 30m
-  timeout: 5m
-
-# default linters are enabled `golangci-lint help linters`
 linters:
-  disable-all: true
+  default: none
   enable:
-  - errcheck
-  - gosec
-  - gosimple
-  - govet
-  - ineffassign
-  - staticcheck
-  - typecheck
-  - unused
-  - stylecheck
```

Comment:
It seems we are removing `stylecheck`. Is v2 not supporting this or any other reason?